### DISCUSSION TITLE
Leaderboards

### DISCUSSION THREAD

**Discussion by RoustyTousty** [PlaceholderAPI/PlaceholderAPI#587] [(2) Support] [answered]
Hello, im looking to make leaderboards using: holographic displays, kite board and placeholder aip.  
I have bin looking for a way to make one for a long time but cant find the aips for it.

**Comment by Andre601** [comment, marked as answer]
It all depends on what kind of leaderboard you want to try creating and if you want to add your own placeholders or use already existing ones.

To my knowledge does HD even have a wiki page showcasing a way to create a leaderboard by youself.

**Reply by RoustyTousty** [reply]
Im trying to make a kill and blocks mined leaderboard.

im just compleatly lost. idk how to make one

**Reply by Andre601** [reply]
I personally suggest to actually asking the developer of HD about what the best approach is.
